openedx/core/djangoapps/user_api/serializers.py

```diff
@@ -84,11 +83,12 @@ def get_writeable_fields(cls):
         return tuple(set(all_fields) - set(cls.get_read_only_fields()))
 
 
-class CountryTimeZoneSerializer(serializers.Serializer):
+class CountryTimeZoneSerializer(serializers.Serializer):  # pylint: disable=W0223
     """
-    Serializer that generates a list of common time zones for country
+    Serializer that generates a list of common time zones for a country
     """
     time_zone = serializers.SerializerMethodField()
 
     def get_time_zone(self, time_zone):
+        """ Returns inputted time zone """
```

Comment:
That's because `time_zone` is a `SerializerMethodField`.  Try something simpler.  Examples here: http://www.django-rest-framework.org/api-guide/fields/